CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased](https://github.com/pusher/chatkit-android/compare/1.0.0...HEAD)
 
+### Fixed
+
+- URL Encoding of room IDs when sending typing indicators
+- Deserialization of Room privacy (always returned false)
+- Deserialization of User customData - was Map<String, String> when other clients were not
+  restricted to string values by the API.
+- Room name is optional when updating rooms
+
+### Added
+
+- Room objects have a custom data field which can hold a map of arbitrary data,
```

Comment:
Worth adding that updates to custom data are propagated in real time?

Reply:
I thought it was implied (all room updates are propagated in real time).

Let's mention it in docs and feature announcement.

Reply:
Checked the docs, it already specifies that members will receive an event with the new room details.